src/engine/enginebuffer.cpp

```diff
@@ -1242,11 +1242,12 @@ void EngineBuffer::postProcess(const int iBufferSize) {
 }
 
 void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
-    if (m_trackSampleRateOld == 0) {
+    if (m_filepos_play == kInitialSamplePosition || m_trackSampleRateOld == 0) {
```

Comment:
use of undeclared identifier 'kInitialSamplePosition'; did you mean 'kInitalSamplePosition'?